retrofit/src/main/java/retrofit/Call.java

```diff
@@ -0,0 +1,10 @@
+package retrofit;
+
+import java.io.IOException;
+
+public interface Call<T> extends Cloneable {
```

Comment:
off topic: and now I'm wondering whether we should have used an interface in OkHttp